fe/lib/components/JobCategorySuggest/JobCategorySuggestChoices.tsx

```diff
@@ -1,13 +1,15 @@
 import { Radio, Stack, Strong, Text } from 'braid-design-system';
-import React, { forwardRef, useState } from 'react';
+import React, { ComponentPropsWithRef, forwardRef, useState } from 'react';
 
 import {
   JobCategory,
   JobCategorySuggestionChoice,
 } from '../../types/seek.graphql';
 import { flattenResourceByKey } from '../../utils';
 
-interface Props {
+interface Tone extends Pick<ComponentPropsWithRef<typeof Radio>, 'tone'> {}
+
+interface Props extends Tone {
```

Comment:
Nit, but one of these may be preferable:

```suggestion
interface Props extends Tone {
  tone: ComponentProps<typeof Radio>['tone']
```

```suggestion
interface Tone extends Pick<ComponentPropsWithRef<typeof Radio>, 'tone'> {}

type Props = Pick<ComponentProps<typeof Radio>, 'tone'> & {
```

Reply:
Oh yeah, I like the first structure better. thanks! :)